modules/core/src/main/scala/org/scalasteward/core/repoconfig/RepoConfig.scala

```diff
@@ -22,7 +22,8 @@ import io.circe.{Decoder, Encoder}
 
 final case class RepoConfig(
     updates: UpdatesConfig = UpdatesConfig(),
-    updatePullRequests: Boolean = true
+    updatePullRequests: Boolean = true,
+    updatesLimit: Option[Int] = None
```

Comment:
Can this be moved to `UpdatesConfig` as `limit` so that it reads as `updates.limit` in `.scala-steward.conf`?